### PR TITLE
doc: add 2016-12-07 meeting minutes

### PR DESCRIPTION
**Pull request by joshgav**
cc @nodejs/ctc @s3ththompson @jkrems @bmeck @renrutnnej

### CODE REVIEW

File: meetings/2016-12-07.md [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+default is important.
+
+@mscdex: Is it the same API as the previous debugger?
+
+@bmeck: Some minor differences due to Console API wrapper, but that’s it AFAIK.
+
+@jkrems: If it’s bundled then it would be hard to remove it, perhaps don’t want
+it in an LTS.
+
+@trott: If it’s like npm than that’s less of a concern.
```
Comment (Trott) [inline meetings/2016-12-07.md]: Nit: `then`

File: meetings/2016-12-07.md [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+
+See also
+[nodejs/diagnostics#67](https://github.com/nodejs/diagnostics/issues/67).
+
+@Trott: Old debugger protocol and CLI debugger is going away.
+
+@ofrobots: Deprecating old CLI debugger, can build a new CLI debugger based on
+new inspector protocol.
+
+PR for removal of V8 Debug API: <https://github.com/nodejs/node/issues/9789>
```
Comment (bnoordhuis) [inline meetings/2016-12-07.md]: https://github.com/nodejs/node/issues/9789 is not a pull request, it's an issue.

**Review by bnoordhuis — DISMISSED** [review]

File: meetings/2016-12-07.md [hunk 3]
```diff
@@ -0,0 +1,313 @@
+# Node Foundation CTC Meeting 2016-12-07
```
Comment (thefourtheye) [inline meetings/2016-12-07.md]: Its officially node.js foundation, right?

Comment (joshgav) [reply meetings/2016-12-07.md]: I suppose so :). Will update, thanks!

**Comment by joshgav** [comment]
Can I get a couple LGTM's? Thanks!

**Review by Trott — APPROVED** [review]
LGTM

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by jasnell — APPROVED** [review]

**Comment by joshgav** [comment]
landed in https://github.com/nodejs/CTC/commit/338a19694c4d9d127987443c8004a1462db06d85

**Pull request closed without merging**